### ISSUE TITLE
[BUG][WEBJS] - Error GET info channel

### ISSUE THREAD

**Issue by Thomas-Pellan** [allburov/whatsapp-http-api#682]
### Describe the bug

GET /api/{session}/channels/{id} returns error (both with channel id and invite id), using a channel as OWNER or SUBSCRIBER

Waha 2024.11.11 WEBJS

```json
{
  "statusCode": 500,
  "timestamp": "2024-12-05T12:01:46.308Z",
  "exception": {
    "stack": "TypeError: Cannot read properties of undefined (reading 'value')\nwindow.WWebJS.getChannelMetadata (pptr:evaluate;Binding.%3Canonymous%3E%20(%2Fapp%2Fnode_modules%2Fwhatsapp-web.js%2Fsrc%2FClient.js%3A226%3A36):505:142)\n    at window.WWebJS.getChannelMetadata (evaluate at Binding.<anonymous> (/app/node_modules/whatsapp-web.js/src/Client.js:226:36), <anonymous>:504:141)\n    at #evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/ExecutionContext.js:388:56)\n    at async ExecutionContext.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/ExecutionContext.js:275:16)\n    at async IsolatedWorld.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/IsolatedWorld.js:100:16)\n    at async CdpFrame.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Frame.js:341:20)\n    at async CdpPage.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Page.js:807:20)\n    at async WebjsClient.getChannelByInviteCode (/app/node_modules/whatsapp-web.js/src/Client.js:1074:16)\n    at async WhatsappSessionWebJSCore.channelsGetChannelByInviteCode (/app/dist/core/engines/webjs/session.webjs.core.js:624:26)\n    at async WhatsappSessionWebJSCore.channelsGetChannel (/app/dist/core/engines/webjs/session.webjs.core.js:621:16)",
    "message": "Cannot read properties of undefined (reading 'value')\nwindow.WWebJS.getChannelMetadata (pptr:evaluate;Binding.%3Canonymous%3E%20(%2Fapp%2Fnode_modules%2Fwhatsapp-web.js%2Fsrc%2FClient.js%3A226%3A36):505:142)",
    "name": "TypeError"
  },
  "request": {
    "path": "/api/default/channels/120363373832647262%40newsletter",
    "method": "GET",
    "body": {},
    "query": {}
  },
  "version": {
    "version": "2024.11.11",
    "engine": "WEBJS",
    "tier": "CORE",
    "browser": "/usr/bin/chromium"
  }
}
```

**Comment by devlikepro** [comment]
Try updating, it's been fixed in `2024.11.11`
https://waha.devlike.pro/docs/overview/changelog/#202411

**Comment by devlikepro** [comment]
Hi! Fixed in 2024.12.1 :crossed_fingers: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
